### PR TITLE
Added language specific page for Rust [resolves #266]

### LINKED ISSUE #266: Programming-Language Specific Pages: Rust
### Prize Bounty
0.3 ETH

### Challenge Description
Let's add resource pages that cater to specific user groups to make Ethereum more approachable to a broader range of people.

Build a dedicated page for Rust developers that includes:
- a brief description of Ethereum
- beginner articles
- intermediate articles
- advanced use patterns
- Rust projects and tools
- Rust communities (e.g. project Gitter channels)

The Python page is a great template to work off:
https://ethereum.org/python/

Also, add a link to the page from the [/developers page "Language Specific Resources" section](https://github.com/ethereum/ethereum-org-website/blob/dev/docs/developers/index.md#language-specific-resources).

### Submission Requirements
The PR must include everything outlined in the challenge description above.

### Submission Deadline
The deadline for submissions is November 11, 2019.

### Judging Criteria
The prize will be awarded to the first valid submission.

### Winner Announcement Date
The submission will be reviewed by November 18. Winner announcements and prize payouts will occur before November 31, 2019.

### PR DESCRIPTION
**Pull request by nabetse00**
Added language specific page for Rust

## Description

Added language specific page for Rust following https://ethereum.org/python/ page as a template

## Related Issue

Resolves #266

## Checklist:

- [x] I have read the contributing guidelines in the project README.
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: docs/rust/index.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+---
+title: Ethereum for Python Developers
+meta:
+  - name: description
+    content: Learn how to develop for Ethereum using rust-based projects and tooling
```
Comment (samajammin) [inline docs/rust/index.md]: `Rust-based`

File: docs/rust/index.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  - property: og:title
+    content: Ethereum for Python Developers
+  - property: og:description
+    content: Learn how to develop for Ethereum using Python-based projects and tooling
```
Comment (samajammin) [inline docs/rust/index.md]: `Rust-based`

File: docs/rust/index.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
+---
+title: Ethereum for Python Developers
```
Comment (samajammin) [inline docs/rust/index.md]: `Rust Developers`

File: docs/rust/index.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+meta:
+  - name: description
+    content: Learn how to develop for Ethereum using rust-based projects and tooling
+  - property: og:title
+    content: Ethereum for Python Developers
```
Comment (samajammin) [inline docs/rust/index.md]: `Rust Developers`

File: docs/rust/index.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+sidebarDepth: 0
+---
+
+# Ethereum for Rust Developers
+
+<div class="featured">Learn how to develop for Ethereum using Rust-based projects and tooling</div><br>
+
+Use Ethereum to create decentralized applications (or "dapps") that utilize the benefits of cryptocurrency and blockchain technology. These dapps can be trustworthy, meaning that once they are deployed to Ethereum, they will always run as programmed. They can control digital assets in order to create new kinds of financial applications. They can be decentralized, meaning that no single entity or person controls them and are nearly impossible to censor.
+
+<img src="https://i.imgur.com/VhoX4LM.png" width="100%" />
```
Comment (samajammin) [inline docs/rust/index.md]: ✂️

**Comment by nabetse00** [comment]
> Thanks for this contribution! You missed a few Python copy/image references from copying the file over but I'll just merge this & make the necessary edits. Next time please note those details 😄

Thanks @samajammin. I'll be more carefoul next time!

**Pull request merged**